libs/utils/env.py

```diff
@@ -420,9 +420,11 @@ def _init_target(self, force = False):
         # Initialize JUNO board
         elif board_name in ('JUNO', 'JUNO2'):
             platform = devlib.platform.arm.Juno()
-            self.nrg_model = juno_energy
             self.__modules = ['bl', 'hwmon', 'cpufreq']
 
+            if board_name == 'JUNO':
+                self.nrg_model = juno_r0_energy
```

Comment:
Aren't we now missing and nrg_model for `JUNO2`?
I.e. look at my previous proposal...

Reply:
It is not missing more than it used to be :-) Since I don't need it at the moment (I always extract it from the target), I'll leave that for another PR and for the person that actually needs the feature.

Reply:
Ok, I see what you mean... and at the end that's exactly the same behaviour we had initially... actually, better, since before we was loading a wrong EM for JunoR2.

One last question, what happen on R0, do we still override the above hardcoded EM if we are able to read them from the target?

Reply:
That's right. Later in the same function, there is this bit of code, so the energy model is only read from the target if not already provided:
```python
        if not self.nrg_model:
            try:
                self._log.info('Attempting to read energy model from target')
                self.nrg_model = EnergyModel.from_target(self.target)
```